core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationSummaryInfo.scala

```diff
@@ -164,6 +172,19 @@ class SingleAppSummaryInfoProvider(val app: ApplicationSummaryInfo)
     }
   }
 
+  // Returns spilled metrics per stage, in the format (stage_id, spilled_metrics)
+  override def getSpillMetricsForShuffleStages: Seq[(Long, Long)] = {
+    // Get stages with positive shuffle read/write
+    app.jsMetAgg.collect { case row if (row.id.contains("stage") &&
+      row.srTotalBytesReadSum + row.swBytesWrittenSum > 0) =>
+        (row.id.split("_")(1).toLong, row.diskBytesSpilledSum + row.memoryBytesSpilledSum)
+    }
```

Comment:
@nartal1 Can you help Cindy with making sure how to pick "shuffles"?
are `row.srTotalBytesReadSum` and `swBytesWrittenSum` exclusive to shuffle operations? IF not, then what is the right conditions to successfully visit all the shuffles?

Reply:
`srTotalBytesReadSum` is the sum of local and remote bytes read in shuffle metrics  across all tasks and `swBytesWrittenSum` is the sum of bytes written for the shuffle across all tasks. The above condition is correct.
So this is getting the spilled metrics for those shuffle stages(which will have either shuffle read or write).